totalopenstation/formats/__init__.py

```diff
@@ -41,14 +41,25 @@ def __init__(self, geom, desc, id=None, **properties):
 
     @g.Feature.geometry.setter
     def geometry(self, value):
+        '''Set the geometry attribute.
```

Comment:
I wasn't really thinking of docstrings for this PR, but OK!

Reply:
It is part of the doc ^^

Reply:
Following https://github.com/steko/totalopenstation/pull/106#issuecomment-549139749 this could be mark as resolved